xbmc/interfaces/Builtins.cpp

```diff
@@ -166,6 +167,7 @@ const BUILT_IN commands[] = {
   { "AlarmClock",                 true,   "Prompt for a length of time and start an alarm clock" },
   { "CancelAlarm",                true,   "Cancels an alarm" },
   { "Action",                     true,   "Executes an action for the active window (same as in keymap)" },
+  { "DialogOK",                   true,   "Shows an OK dialog on screen, specify header, then message." },
```

Comment:
DialogOK seems fine to me. With "alert"  I would expect to also have a alert icon. If they want an alert they can use the kaitoast which has icons